options/components/webBottomContent/WebBottomContent.jsx

```diff
@@ -4,31 +4,69 @@ import { WebSearchContext } from "../../context/WebSearchContext";
 
 export default function WebBottomContent() {
   return (
-    <div className="w-full h-[calc(100vh-200px)] overflow-hidden max-w-5xl mx-auto my-0">
+    <div className="w-full h-[calc(100vh-200px)] overflow-hidden lg:max-w-5xl lg:px-0 md:max-w-screen-md sm:max-w-screen-sm px-2 mx-auto my-0">
       <WebUrlNewList />
     </div>
   );
 }
 
 function WebUrlNewList() {
-  const { bookmarkList } = useContext(WebSearchContext);
+  const { bookmarkList, hasSearchResult } = useContext(WebSearchContext);
 
+  function faviconURL(u) {
+    const url = new URL(chrome.runtime.getURL("/_favicon/"));
+    url.searchParams.set("pageUrl", u);
+    url.searchParams.set("size", "32");
+    return url.toString();
+  }
+  console.log(hasSearchResult);
   return (
-    <div className="w-full h-[calc(100vh-200px)] overflow-y-auto">
+    <div className="w-full overflow-y-auto grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-2 gap-4">
       {bookmarkList.map((url, index) => (
         <div
-          className="w-full flex"
+          className="w-full min-h-10 flex bg-white rounded-lg p-3 hover:bg-gray-300 hover:shadow-lg duration-100"
           key={index}
         >
           <a
-            className="w-full max-w-[calc(100%-10px)] flex-grow overflow-hidden text-ellipsis whitespace-nowrap"
+            className="w-full max-w-[calc(100%-0px)] flex-grow overflow-hidden text-ellipsis whitespace-nowrap font-bold"
             href={`${url.url}`}
             target="_blank"
           >
+            <img
+              className="mr-2 inline-block w-3 h-3"
+              src={faviconURL(url.url)}
+            />
             {url.title}
+            {hasSearchResult ? <HighlightKeyword url={url} /> : ""}
```

Comment:
```suggestion
            {hasSearchResult && <HighlightKeyword url={url} />}
```
false인 상황에 공백이 표시되어야 한다면 제 개인적인 의견으로는 다음과 같은 상황에서는 삼항연산자보다 단축평가를 사용하는것도 가독성을 향상시키는데 좋은 방법이라고 생각합니다.